profile/README.md

```diff
@@ -1,12 +1,16 @@
 ![logo](https://github.com/confidential-containers/.github/blob/main/coco_logo.png)
 
-## Welcome to confidential-containers
+## Welcome to Confidential Containers
 
 Confidential Containers is an open source community working to enable cloud native confidential 
 computing by leveraging 
 [Trusted Execution Environments](https://en.wikipedia.org/wiki/Trusted_execution_environment) to 
 protect containers and data.
 
+**Release v0.1.0 is Live!**
+See [Release Notes](https://github.com/confidential-containers/documentation/blob/main/releases/v0.1.0.md)
+or [Quickstart Guide](https://github.com/confidential-containers/documentation/blob/main/quickstart.md)
```

Comment:
Maybe link the quickstart.md on the tag rather than `main` as the file on main can get updated after 0.1.0

Reply:
Ok, I can update this once we tag the docs.